script/Ciel/Day 9/09_01_CIEL09_2.txt

```diff
@@ -252,22 +251,23 @@ This is a weapon that punishes lia... no, vampires by driving a pile through the
 [sha:ea751656faa4b35e0ebc23dad8c20942828f6045]{
 -- Page 116, Offset 22435.
 -- 何かいま、背筋が凍るどころか背骨ごと打ち砕かれそうな言い間違いがあった気がする。
-I feel that she just made a slip of the tongue, one that promises not just to chill my spine but rather to crush it.
+//man this line sucks
+I feel like she just made a slip of the tongue. Rather than sending a chill down my spine, that slip of the tongue felt like it was going to crush it.
```

Comment:
"it might crush it" maybe? that "it" at the end is definitely a bit awkward but i'm having trouble rephrasing it